ddprof-exporter/src/lib.rs

```diff
@@ -1,6 +1,9 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
 
+#[macro_use]
+extern crate lazy_static;
```

Comment:
Does this need to be in this file? I don't see it used anywhere.

Reply:
Hum, seems like I have been mislead by lazy_static crate documentation that refers to the "old" way of importing macros (and needs to be done in crate root file).
With 2018 edition, I should be able to replace that with a `use lazy_static::lazy_static` in containerid.rs  (https://veykril.github.io/tlborm/decl-macros/minutiae/import-export.html).